src/dataDisplay/personCoreMilestones/personCoreMilestones.jsx

```diff
@@ -140,17 +187,24 @@ const useStyles = makeStyles((theme) => {
             padding: `0 ${columnHorizontalPadding}px`,
             width: 'auto !important',
         },
-        congregationDateHeading: {
-            color: palette.text.secondary,
+        congregationDateTypography: {
+            color: ({ backgroundTransparent }) => (backgroundTransparent ?
+                palette.text.contrastText :
+                palette.text.secondary
+            ),
         },
         firstContactDateColumn: {
-            flexGrow: 1,
-            padding: `0 ${columnHorizontalPadding}px`,
-            textAlign: 'right',
+            padding: ({ isMobile }) => (isMobile ?
+                '5px 15px 5px 15px !important' :
+                '10px 15px 10px 5px !important'
```

Comment:
_Not a Blocker_  Is `!important` our only recourse here?  And/or we're okay with it?

Reply:
Yes, it was the only way to make it work.